samples/places-autocomplete-addressform/src/style.scss

```
@@ -19,44 +19,72 @@
 /* [START maps_places_autocomplete_addressform] */
 @include meta.load-css("../../../shared/scss/default.scss");
 
-#locationField,
-#controls {
-  position: relative;
-  width: 480px;
-}
-#autocomplete {
-  position: absolute;
-  top: 0px;
-  left: 0px;
-  width: 99%;
-}
-.label {
-  text-align: right;
-  font-weight: bold;
-  width: 100px;
-  color: #303030;
-  font-family: "Roboto", Arial, Helvetica, sans-serif;
-}
-#address {
-  border: 1px solid #000090;
-  background-color: #f0f9ff;
-  width: 480px;
-  padding-right: 2px;
-}
-#address td {
-  font-size: 10pt;
-}
-.field {
-  width: 99%;
-}
-.slimField {
-  width: 80px;
-}
-.wideField {
-  width: 200px;
-}
-#locationField {
-  height: 20px;
-  margin-bottom: 2px;
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  color: rgb(104, 104, 104);
+}
+
+form {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 400px;
+  margin: 20px auto;
+  padding: 4rem 0;
+}
+
+input {
+  width: 100%;
+  height: 1.2rem;
+  margin-top: 0;
+  padding: 0.5em;
+  border: 0;
+  border-bottom: 2px solid gray;
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+}
+
+input:focus {
+  border-bottom: 4px solid black;
+}
+
+.title {
+  font-weight: 500;
+}
+
+.formlabel {
+  width: 100%;
+  padding: 0.5em;
+}
+
+.fullfield {
+  flex: 400px;
+  margin: 15px 0;
+}
+
+.slimfield-left {
+  flex: 1 150px;
+  margin: 15px 15px 15px 0px;
+}
+
+.slimfield-right {
+  flex: 1 150px;
+  margin: 15px 0px 15px 15px;
+}
+
+.myButton {
+	background-color:#000;
+	border-radius:6px;
+	color:#fff;
+  margin: 10px;
+	padding:6px 24px;
+	text-decoration:none;
+}
+.myButton:hover {
```

```suggestion
.my-button:hover {
```
since scss, can also be moved into .my-button block with just `:hover {`